app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -196,6 +196,13 @@ def user_data_supported?
       true
     end
 
+    def console(uuid)
+      vm = find_vm_by_uuid(uuid)
+      { 'output' =>  vm.serial_port_output,
+        'timestamp' => Time.now.utc
```

Comment:
Doesn't the output coming with it's own timestamp?
As this doesn't need to be precise if there is some delay on the line.
It is not big issue, but if there would be way to get the output with timestamp, it would be :heart_decoration:

Reply:
No, I have checked but Google serial output API doesn't return any timestamp.
Response by respective API is as follows - 
```
{
  "contents": string,
  "start": string,
  "next": string,
  "selfLink": string,
  "kind": string
}
```
Behind the scene, fog-google users [getSerialPortOutput](https://cloud.google.com/compute/docs/reference/rest/v1/instances/getSerialPortOutput) API